project/minchuncho/README.rst

```diff
@@ -0,0 +1,73 @@
+==================================================
+SewUP
+==================================================
+Let's have the Stackelberg game sewed up.
+
+Basic Information
+=================
+**SewUP** is a tool for sorting out the competition in a Stackelberg game in certain cases.
+
+Problem to Solve
+=================
+A Stackelberg model is a strategic game which consists of a leader and a group of followers, who compete over a quantity while maximizing their profits or minimizing their costs. In the model, the leader acts first and the followers respond afterwards, which goes on and on until everyone settles.  
+However, whether or not the game will converge is up in the air. Gladly, such games take advantages of **backward induction** where the leader predicts followers’ best responses under a given strategy of his. If luckily, those best responses can be rewritten as functions of the leader’s strategy, then the leader can substitute those best responses back to his utility (or cost) function and find the extremum. Obviously, the game will stop with **just one back-and-forth**, which is quite efficient. Note that this approach favors the leader, which is called “first mover’s advantage.”
+
+.. Since the whole process is determined by the leader, it is like the leader is trying to bring the situation to his/her favorable conclusion, hence the project is named “SewUP.”
+
+Prospective Users
+==================
+For those researchers who study resource allcation and incentive mechanisms, especially one-to-many Stackelberg game with certain formats of utility functions, their problems can be examined or even solved by the tool.
```

Comment:
It will help specify the system clearer to describe the motivation for the researchers to study the problems.

Reply:
I added some extra content to it and hope it's clearer now.